tests/worker/backends/test_backend.py

```diff
@@ -0,0 +1,46 @@
+import types
+
+import pytest
+
+from gpustack.worker.backends.custom import CustomServer
+
+
+@pytest.mark.parametrize(
+    "image_name, container_registry, expect_image_name",
```

Comment:
Can you add cases:
1. image without a tag, e.g., `foo/bar`
2. image without namespace, e.g., `ubuntu` or `ubuntu:24.04`